santoku/utils/test/test_url_handler.py

```diff
@@ -58,6 +58,9 @@ def test_string_subdomain_is_processed_as_expected(input_subdomain, expected_sub
         ("", (None, [], None, None, None)),
         ("https:///integration/ckeditor", ("https", [], None, None, "integration/ckeditor")),
         ("/quizzesproxy/quizzes/service", (None, [], None, None, "quizzesproxy/quizzes/service")),
+        ("example.com/path", (None, [], "example", "com", "path")),
```

Comment:
question: Does it make sense to test with strings containing uppercase/lowercase in domain/path?